packages/dialect-react-ui/components/Broadcast/BroadcastForm.tsx

```diff
@@ -0,0 +1,94 @@
+import { Dapp, useDappAddresses } from '@dialectlabs/react-sdk';
+import clsx from 'clsx';
+import { useState } from 'react';
+import { Button, ValueRow } from '../common';
+import { H1, Input, P, Textarea } from '../common/preflighted';
+import { useTheme } from '../common/providers/DialectThemeProvider';
+
+interface BroadcastFormProps {
+  dapp: Dapp;
+}
+
+function BroadcastForm({ dapp }: BroadcastFormProps) {
+  const { addresses, isFetching: isFetchingAddresses } = useDappAddresses();
+  const { textStyles, colors, outlinedInput, textArea } = useTheme();
+  // Consider moving error handling to the useDapp context
+  const [error, setError] = useState<Error | null>(null);
+  const [isSending, setIsSending] = useState(false);
+  const [title, setTitle] = useState('');
+  const [message, setMessage] = useState('');
+
+  const usersCount = addresses?.length || 0;
```

Comment:
Dapp address is actually not a user
Users count = set of unique wallets, associated with enabled dapp addresses, associated with verified addresses

Reply:
Got it, fixed!